sdcm/utils/docker_remote.py

```diff
@@ -53,17 +54,19 @@ def get_log(self):
         return self.node.remoter.run(f"docker logs {self.docker_id}").stdout.strip()
 
     def run(self, cmd, *args, **kwargs):
-        return self.node.remoter.run(f'docker exec {self.docker_id} /bin/bash -c "{cmd}"', *args, **kwargs)
+        cmd = cmd.replace('"', '\\"')
+        cmd = shell_script_cmd(cmd, shell_cmd='/bin/sh -c')
+        return self.node.remoter.run(f'docker exec {self.docker_id} {cmd}', *args, **kwargs)
 
     def kill(self):
         return self.node.remoter.run(f"docker rm -f {self.docker_id}", verbose=False, ignore_status=True)
 
     def send_files(self, src, dst, **kwargs):
         self.node.remoter.send_files(src, src, **kwargs)
-        self.node.remoter.run(f"docker cp {src} {self.docker_id}:{dst}", verbose=False, ignore_status=True)
+        self.node.remoter.run(f"docker cp {src} {self.docker_id}:{dst}", verbose=True, ignore_status=False)
```

Comment:
Why was changed ignore_status=False, if for some reason copy  failed, it will failed all test.

Reply:
I've changed it for debugging, since it silently failing that copy.

and in general it should be, it should be default to `verbose=True and ignore_status=False`, like the defaults in `send_files` and `run`

Reply:
fixed